QUEST_LV_0100.tsv

```diff
@@ -5248,12 +5248,12 @@ QUEST_LV_0100_20150317_005247	Read the last epitaph in Apati Cliff
 QUEST_LV_0100_20150317_005248	{memo X}It is the last clue prepared by King Zachariel. Read the epitaph in Apati Cliff and get the fifth clue. 
 QUEST_LV_0100_20150317_005249	{memo X}Got the fifth clue. Go back and talk to Rexipher.
 QUEST_LV_0100_20150317_005250	Protect the Mausoleum Stone Lantern
-QUEST_LV_0100_20150317_005251	The guardian statue says the flow of spell in the Mausoleum will be twisted if the stone lanterns are destroyed. Protect the stone lanterns from demons. 
+QUEST_LV_0100_20150317_005251	The Protector Statue says the flow of spell in the Mausoleum will be twisted if the stone lanterns are destroyed. Protect the stone lanterns from demons. 
 QUEST_LV_0100_20150317_005252	Follow Rexipher to Mausoleum 3rd Floor
-QUEST_LV_0100_20150317_005253	Rexipher destroyed the stone lantern and disappeared to the deeper area. Follow him to the 3rd Floor. 
-QUEST_LV_0100_20150317_005254	Defeat the contaminated guardian
-QUEST_LV_0100_20150317_005255	The twisted spell is causing the guardian to attack both enemies and allies. Defeat the contaminated guardians.
-QUEST_LV_0100_20150317_005256	Defeat the Echad that can't identify friend or enemy
+QUEST_LV_0100_20150317_005253	Rexipher destroyed the stone lantern and disappeared to a deeper area. Follow him to the 3rd Floor. 
+QUEST_LV_0100_20150317_005254	Defeat the corrupted protector
+QUEST_LV_0100_20150317_005255	A twisted spell is causing the protector to attack both enemies and allies. Defeat the corrupted protector.
```

Comment:
The Guardians/protectors are a collective term used to describe any of the mobs found there. Think of it as Secret Service kind of group